src/components/Snackbar/SnackbarContext.tsx

```diff
@@ -131,24 +139,40 @@ export const SnackbarProvider: React.FC = ({ children }) => {
       return null;
     }
 
+    const getPositionProps = (position: keyof typeof positionStyles) => {
+      return positionStyles[position];
+    };
+
+    const listOfPositions = Object.keys(positionStyles) as (keyof typeof positionStyles)[];
     return ReactDOM.createPortal(
-      <Flex
-        position="fixed"
-        bottom={3}
-        right={6}
-        flexDirection="column"
-        justifyContent="center"
-        alignItems="flex-end"
-        zIndex={9999}
-      >
-        {transitions.map(({ item: { id, ...snackbarPublicProps }, key, props: styles }) => (
-          <animated.div key={key} style={styles}>
-            <Box pt={3} ref={ref}>
-              <Snackbar destroy={() => removeSnackbar(id)} {...snackbarPublicProps} />
-            </Box>
-          </animated.div>
+      <>
+        {listOfPositions.map(position => (
+          <Flex
+            key={position}
+            position="fixed"
+            flexDirection="column"
+            justifyContent="center"
+            alignItems="flex-end"
+            zIndex={9999}
+            {...getPositionProps(position)}
+          >
+            {transitions
+              .filter(snackbar =>
+                // handles multiple toasts in different positions
+                snackbar.item.position
+                  ? snackbar.item.position === position
+                  : position === 'bottom-left'
```

Comment:
nit: not 100% sure what happens here, but I trust you. If it's not too much trouble, add an even more detailed comment to help future readers

Reply:
Added it! Thanks :)